src/schema/sale/__tests__/index.test.js

```diff
@@ -175,7 +175,7 @@ describe("Sale type", () => {
     })
   })
 
-  describe("sale_artworks_connection", async () => {
+  describe("sale_artworks_connection", () => {
```

Comment:
I fixed some of these yesterday in reaction. I think we should find/write a lint rule to catch these.

Reply:
Perhaps, yeah